crowbar_framework/spec/models/api/upgrade_spec.rb

```diff
@@ -53,25 +54,25 @@
   context "with a successful status" do
     it "checks the status" do
       allow(Crowbar::Sanity).to receive(:sane?).and_return(true)
-      allow_any_instance_of(Api::Crowbar).to(
+      allow(Api::Crowbar).to(
         receive(:features).and_return([])
       )
 
+      allow(Api::Crowbar).to(
+        receive(:addons).and_return(
+          ["ha"]
+        )
+      )
+      allow(Api::Crowbar).to(
+        receive(:ha_presence_check).and_return({})
+      )
+
       expect(subject.class).to respond_to(:status)
       expect(subject.class.status).to be_a(Hash)
       expect(subject.class.status.deep_stringify_keys).to eq(upgrade_status)
     end
   end
 
-  context "with a successful maintenance updates check" do
```

Comment:
Yes, I just remembered that and I'm looking at 
https://github.com/crowbar/crowbar-ha/pull/147 right now. But in general I feel that a lot of time is being wasted currently within the review process due to lack of commit hygiene. I strongly believe we should adhere to the same [standards in upstream OpenStack](https://wiki.openstack.org/wiki/GitCommitMessages). I'll raise it in the next standup.